src/pgrn-snippet-html.c

```diff
@@ -120,18 +119,19 @@ PGrnSnipExec(grn_obj *snip, text *target, ArrayType **snippetArray)
 }
 
 /**
- * pgroonga.snippet_html(target text, keywords text[]) : text[]
+ * pgroonga_snippet_html(target text, keywords text[], unsigned int width) : text[]
  */
 Datum
 pgroonga_snippet_html(PG_FUNCTION_ARGS)
 {
 	const char *tag = "[snippet-html]";
 	text *target = PG_GETARG_TEXT_PP(0);
 	ArrayType *keywords = PG_GETARG_ARRAYTYPE_P(1);
+	unsigned int width = PG_GETARG_INT32(2);
```

Comment:
Could you add a check for `PG_GETARG_INT32(2) >= 0`?
We can use `ereport(ERROR, (errcoe(ERRCODE_INVALID_PARAMETER_VALUE), errmsg("...")))` to report an error for invalid input.

Reply:
Added